docs/api.md

```diff
@@ -519,6 +520,10 @@ Provide credentials for [http authentication](https://developer.mozilla.org/en-U
 
 To disable authentication, pass `null`.
 
+#### page.bringToFront()
```

Comment:
here return value needs to be declared:

```
#### page.bringToFront()

- returns: <[Promise]>

Brings page to front (activates tab).
```

You can check that your CL passes all lints/checks by running `npm test` locally.